aspnetcore/blazor/class-libraries.md

```diff
@@ -12,6 +12,8 @@ uid: blazor/class-libraries
 
 By [Simon Timms](https://github.com/stimms)
 
+[!INCLUDE[](~/includes/blazorwasm-preview-notice.md)]
```

Comment:
I think we can remove this one. There isn't anything in this article that is Blazor WebAssembly specific.